Src/SmtpServer/Protocol/MailCommand.cs

```diff
@@ -13,7 +13,7 @@ namespace SmtpServer.Protocol
     /// </summary>
     public sealed class MailCommand : SmtpCommand
     {
-        public const string Command = "MAIL";
+        internal const string Command = "MAIL";
```

Comment:
all these should really stay public too...was their a specifc reason you want them internal?